torch/_inductor/codegen/triton.py

```diff
@@ -863,8 +862,12 @@ def load(self, name: str, index: sympy.Expr):
     def store(self, name, index, value, mode=None):
         var = self.args.output(name)
         index, mask = self.indexing(index, dense_indexing=True)
+
+        # Keep the variable in cache if we are going to reuse it
+        ep_str = "first" if name in self.current_node.last_usage else "last"
```

Comment:
do we need store eviction policy? When we are writing and then reading the data, we typically need to go all the way to gmem, otherwise we would just leave it as an intermediate variable

Reply:
In the current triton master (MLIR) there's no `eviction_policy` arg to store, so let's hold off on this.

Reply:
ag, they said it'd be BC aaaand here we are. Leaving it as a comment in case it's relevant in the future.